MaxText/train.py

```diff
@@ -289,6 +289,12 @@ def train_loop(config, state=None):
   max_logging.log(f"number parameters: {num_model_parameters/10**9:.3f} billion")
   per_device_tflops = calculate_training_tflops(num_model_parameters, config)
 
+  # Write train config params and number of model params to tensorboard
```

Comment:
Is it ok to write the train config params to tensorboard by default or should it be put behind a flag?

Reply:
Seems fine.